doc/dependencies.md

```diff
@@ -19,8 +19,8 @@ Bitcoin Core requires one of the following compilers.
 
 | Dependency | Releases | Version used | Minimum required |
 | --- | --- | --- | --- |
-| CMake | [link](https://cmake.org/) | N/A | [3.22](https://github.com/bitcoin/bitcoin/pull/30454) |
 | [Boost](../depends/packages/boost.mk) | [link](https://www.boost.org/users/download/) | [1.81.0](https://github.com/bitcoin/bitcoin/pull/26557) | [1.73.0](https://github.com/bitcoin/bitcoin/pull/29066) |
+| CMake | [link](https://cmake.org/) | [3.24.2](https://github.com/bitcoin/bitcoin/pull/30730) | [3.22](https://github.com/bitcoin/bitcoin/pull/30454) |
```

Comment:
In commit fefe8fc3c38701bdc3a3b1c6e4d0c8885f94e75f ISTM the CMake entry shouldn't change in this commit, apart from ordering alphabetically (why does it change here)

Reply:
ah yes messed up the rebase..fixed now.